app/assets/raw/assets/footer/ama-logo-small.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 133.7 32.6" enable-background="new 0 0 133.7 32.6"><style type="text/css">.st0{fill:#FFFFFF;} .st1{fill:#FFFFFF;stroke:#FFFFFF;stroke-miterlimit:10;} .st2{fill:#FFFFFF;stroke:#FFFFFF;stroke-width:0.3594;} .st3{fill:#00539B;} .st4{fill:#DA291C;} .st5{fill-rule:evenodd;clip-rule:evenodd;fill:#DA291C;}</style><path d="M62 2.4h12.7l6.4 20h.1l6.7-20H100v30.1h-7.9l.1-23.2h-.1l-7.7 23.2H77L69.8 9.4h-.1l.2 23.2H62V2.4zm-21.2 0h9.8l11.6 30.1h-8.8l-2.2-6.4H39.8l-2.4 6.4h-8.2L40.8 2.4zm4.9 6.2l-4 11.8h7.6L45.7 8.6zm65.6-6.2h9.9l11.6 30.1H124l-2.2-6.4h-11.3l-2.4 6.4h-8.2l11.4-30.1zm4.9 6.2l-4 11.8h7.6l-3.6-11.8z" class="st0"/><path d="M1 13.7C.9 6.5 9.4.7 20 .7c10.5 0 19.1 5.8 19.2 13 .1 7.2-8.4 13-19 13-10.5 0-19.1-5.8-19.2-13z" class="st1"/><path d="M36.6 8.2c2 2.5 2.5 6.5.9 9.3-3.6 6.2-10.3 8.1-17 8.4-7.1 0-13.9-2-17.8-8-1.5-2.7-1.2-6.7.6-9.2 5.3-6.9 14-8.2 22.3-6.8 4.2 1.1 8.1 2.7 11 6.3z" class="st2"/><path d="M36.6 8.2c2 2.5 2.5 6.5.9 9.3-3.6 6.2-10.3 8.1-17 8.4-7.1 0-13.9-2-17.8-8-1.5-2.7-1.2-6.7.6-9.2 5.3-6.9 14-8.2 22.3-6.8 4.2 1.1 8.1 2.7 11 6.3z" class="st3"/><path d="M34.4 10.1c1 1.4 1.5 3.5 1 5.2-1.8 5.1-7.4 7-12.2 7.7-6.1.7-12.3-.3-16.7-4.5-1.7-1.8-2.5-4.4-1.6-6.8 2.5-5.3 8.3-6.9 13.6-7.5 6.2-.1 12 1 15.9 5.9z" class="st0"/><path d="M24.5 12.8l.3.8c-.5 1.6-1 3.2-1.4 4.8h2.9l1.2 2.7-3.3.9c-.6-1.5-1.1-3.1-1.6-4.6H17c-.5 1.4-1 2.9-1.4 4.4-1.1-.1-2.2-.4-3.4-.7.2-.8.5-1.6.8-2.3 1.6-.8 2.9-2.5 2.9-4.4h-1.4l3.2-9.5c1.4 0 2.5-.1 3.9-.1l2.9 8zm9 5.2h-2.7c-.2-.7-.4-1.3-.6-2H27c-.2.6-.3 1.3-.4 2h-2.7L27 7.6h3.1L33.5 18zM14.8 9.5c.3.5.5 1.1.5 1.8h-2.6c-.1-.4-.1-.8-.5-1.1-.4-.3-1-.4-1.5-.2-1.6.8-1.4 2.7-1.3 4.4.2.8.5 1.9 1.4 2.1.5 0 1 0 1.4-.3.4-.4.5-.9.7-1.3h2.6c-.2 1.8-1.5 3.4-3.3 3.8-1.6.2-3.2-.2-4.2-1.4-1.6-1.9-1.8-4.7-.8-6.9C7.9 9 9.4 7.8 11 7.7c1.5-.1 2.8.5 3.8 1.8z" class="st4"/><path d="M21.6 14.7h-3.8l1.8-6.1c.7 2 1.4 4 2 6.1zm6-.9c.2-1.1.5-2.2.8-3.3l1 3.3h-1.8z" class="st0"/><path d="M33.9 16.2c.3 0 .6-.3.6-.6s-.3-.6-.6-.6-.6.3-.6.6.3.6.6.6zm-.5-.6c0-.3.2-.5.5-.5s.5.2.5.5-.2.5-.5.5-.5-.2-.5-.5zm.4 0h.1l.2.3h.1l-.2-.3c.1 0 .2-.1.2-.2s-.1-.2-.2-.2h-.3v.7h.1v-.3zm0-.1v-.2h.1c.1 0 .2 0 .2.1s-.1.1-.2.1h-.1z" class="st5"/></svg>
```

Comment:
do we need all this svg code? could we just put it in as the image path?

Reply:
@sineade I just moved the .svg to a new directory, so Im not sure why it's showing up in git as code. It's probably just reading what's in the .svg file

Reply:
@sineade I just moved the .svg to a new directory, so Im not sure why it's showing up in git as code. It's probably just reading what's in the .svg file